core/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN /home/pi/tools/dnsmasq/bootstrap.sh
 RUN /home/pi/tools/filebrowser/bootstrap.sh
 RUN /home/pi/tools/linux2rest/bootstrap.sh
 RUN /home/pi/tools/nginx/bootstrap.sh
+RUN /home/pi/tools/logviewer/bootstrap.sh
```

Comment:
The list is in alphabetical order, can you fix logviewer position ?

Reply:
done